src/swagger/openapi.json

```diff
@@ -134,6 +134,17 @@
           "logout"
         ],
         "summary": "Log a user out",
+        "parameters": [
+          {
+            "in": "header",
+            "name": "token",
```

Comment:
The header name is `authentication-header` not `token`.

>2. The token should be added as an authentication-header header to a request, wherever applicable

Reply:
Thanks for the catch!